rs/node_rewards/rewards_calculation/src/rewards_calculator_results.rs

```diff
@@ -12,7 +11,7 @@ use std::fmt::Display;
 pub type XDRPermyriad = Decimal;
 pub type Percent = Decimal;
 
-#[derive(Clone, Debug, PartialEq, Hash, PartialOrd, Ord, Eq, Copy, CandidType)]
+#[derive(Clone, Debug, PartialEq, Hash, PartialOrd, Ord, Eq, Copy)]
 pub struct DayUtc(UnixTsNanos);
```

Comment:
If this is going to be returned in an API, you will want to redefine it in the api crate, so that something needing the API types to interact with the canister doesn't need to import the whole canister as a cargo dependency.